code/modules/paperwork/pen/multi_pen.dm

```diff
@@ -1,20 +1,31 @@
 /obj/item/pen/multi
-	name = "multicoloured pen"
-	desc = "It's a pen with multiple colors of ink!"
-	var/selectedColor = 1
-	var/colors = list("black","blue","red","green")
-	var/color_descriptions = list("black ink", "blue ink", "red ink", "green ink")
-	var/color_icons = list(
+	name                    = "multicoloured pen"
+	desc                    = "It's a pen with multiple colors of ink!"
+	pen_quality             = TOOL_QUALITY_MEDIOCRE
+	var/colour_idx          = 1
+	var/stroke_colours      = list("black", "blue", "red", "green")
+	var/stroke_colour_names = list("black", "blue", "red", "green")
+	var/colour_icons        = list(
 		'icons/obj/items/pens/pen.dmi',
 		'icons/obj/items/pens/pen_blue.dmi',
 		'icons/obj/items/pens/pen_red.dmi',
 		'icons/obj/items/pens/pen_green.dmi',
 	)
 
+/obj/item/pen/multi/Initialize(ml, material_key)
+	. = ..()
+	change_colour(colour_idx)
+
+/obj/item/pen/multi/make_pen_description()
+	desc = "It's a [istype(material)?"[material.name] ":""]pen with multiple colors of ink! Its currently set to [stroke_colour_name] [medium_name]."
```

Comment:
`It's`

Reply:
Hang on I think I'm having a brainfart, not sure if any material name would need 'an'

Reply:
I guess maybe osmium?

Reply:
Ahah yes.